tailwind.config.js

```diff
@@ -76,81 +75,94 @@ module.exports = {
 			0: '0',
 			px: '1px',
 			pico: '1rem',
-			nano: '3rem',
+			nano: '4rem',
 			micro: '8rem',
 			thinnest: '12rem',
 			thinner: '16rem',
 			thin: '20rem',
+			'thin+': '24rem',
```

Comment:
"+" items in the scale should not exist. They are a last resort if more values are needed but should be added per project.

Reply:
That's right, my copy/paste mistake! Will remove.